src/state/queries/preferences/moderation.ts

```diff
@@ -86,54 +88,54 @@ export const CONFIGURABLE_LABEL_GROUPS: Record<
 > = {
```

Comment:
@pfrazee More of a moderation question -- do we want to risk our intention of these labels being mistook by allowing the title and subtitle to be localized strings?